admin/class-help-center.php

```diff
@@ -7,50 +7,110 @@
-	 */
-	private $help_center_items = array();
+	/** @var WPSEO_Option_Tab[] $tab */
+	private $tabs;
 
-	/**
-	 * @var String $group_name
-	 */
-	private $group_name;
+	/** @var string Mount point in the HTML */
+	private $identifier = 'yoast-help-center-container';
 
-	/**
-	 * @var WPSEO_Option_Tab $tab
-	 */
-	private $tab;
+	protected $help_center_items = array();
 
 	/**
 	 * WPSEO_Help_Center constructor.
 	 *
-	 * @param String           $group_name The name of the group of the tab the helpcenter is on.
-	 * @param WPSEO_Option_Tab $tab        The name of the tab the helpcenter is on.
+	 * @param WPSEO_Option_Tabs $tabs Currently displayed tabs.
 	 */
-	public function __construct( $group_name, $tab ) {
-		$this->group_name = $group_name;
-		$this->tab        = $tab;
+	public function __construct( WPSEO_Option_Tabs $tabs ) {
+		$this->tabs = $tabs;
+	}
 
-		$this->add_video_tutorial_item();
-		$this->add_kb_search_item();
+	/**
+	 *
+	 */
+	public function localize_data() {
 		$this->add_contact_support_item();
+		$this->enqueue_localized_data( $this->format_data( $this->tabs->get_tabs() ) );
 	}
 
 	/**
-	 * Add the knowledge base search help center item to the help center.
+	 * @param WPSEO_Option_Tab[] $tabs
+	 *
+	 * @return array
 	 */
-	private function add_kb_search_item() {
-		$kb_help_center_item = new WPSEO_Help_Center_Item(
-			'knowledge-base',
-			__( 'Knowledge base', 'wordpress-seo' ),
+	protected function format_data( array $tabs ) {
+		$formatted_data = array( 'tabs' => array() );
+
+		foreach ( $tabs as $tab ) {
+			$formatted_data['tabs'][ $tab->get_name() ] = array(
+				'label'    => $tab->get_label(),
+				'videoUrl' => $tab->get_video_url(),
+				'id'       => $tab->get_name(),
+			);
+		}
+
+		$active_tab = $this->tabs->get_active_tab();
+		$active_tab = null === $active_tab ? $tabs[0] : $active_tab;
+
+		$formatted_data['mountId']    = $this->identifier;
+		$formatted_data['initialTab'] = $active_tab->get_name();
+
+		$is_premium = WPSEO_Utils::is_yoast_seo_premium();
+
+		// Will translate to either empty string or "1" in localised script.
+		$formatted_data['isPremium']     = $is_premium;
+		$formatted_data['pluginVersion'] = WPSEO_VERSION;
+
+		// Open HelpScout on activating this tab ID:
+		$formatted_data['premiumSupportId'] = $is_premium ? 'contact-support' : '';
+
+		$formatted_data['translations'] = self::get_translated_texts();
+
+		$formatted_data['videoDescriptions'] = array(
 			array(
-				'content'        => '<div class="wpseo-kb-search"></div>',
-				'view_arguments' => array( 'identifier' => $this->tab->get_name() ),
+				'title' => __( 'Need some help?', 'wordpress-seo' ),
+				'description' => __( 'Go Premium and our experts will be there for you to answer any questions you might have about the setup and use of the plugin.', 'wordpress-seo' ),
+				'link' => 'https://yoa.st/seo-premium-vt?utm_content=' . WPSEO_VERSION,
```

Comment:
Please use the `WPSEO_Shortlinker::get` method to add the `utm_content` to the shortlink.